apps/mobile/src/app/(home)/settings/notifications/email.tsx

```diff
@@ -0,0 +1,58 @@
+import { useRef } from 'react';
+import { ScrollView, TouchableOpacity } from 'react-native-gesture-handler';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { EmailAddressSheet } from '@/features/settings/email-address-sheet';
+import { t } from '@lingui/macro';
+import { useTheme } from '@shopify/restyle';
+
+import {
+  Avatar,
+  Box,
+  ChevronRightIcon,
+  EmailIcon,
+  Flag,
+  ItemLayout,
+  SheetRef,
+  Theme,
+} from '@leather.io/ui/native';
+
+export default function SettingsNotificationsEmailScreen() {
+  const { bottom } = useSafeAreaInsets();
+  const emailAddressSheetRef = useRef<SheetRef>(null);
+  const theme = useTheme<Theme>();
+
+  return (
+    <Box flex={1} backgroundColor="ink.background-primary">
+      <ScrollView
```

Comment:
Not for this PR but we seem to repeat this `<ScrollView` with these props a lot (5 times). We could look into a layout for it?

Reply:
I forgot I created a `SettingsScreenLayout` to use here, so will do. Thx for reminder.